.gitignore

```diff
@@ -3,12 +3,6 @@ __pycache__/
 *.py[cod]
 *$py.class
 
-# Auto-generated python files from the protocol buffer compiler
-py/**/*_pb2.py
-py/**/*_pb2.pyi
-py/**/*_pb2_grpc.py
-py/**/*_pb2_grpc.pyi
-
```

Comment:
Mmm, interesting that this got removed, I think it should stay, because these files will be generated as soon as you use `pip install -e .`, and you don't want to add them to the repo accidentally. We should add this too to repo-config:

* https://github.com/frequenz-floss/frequenz-repo-config-python/issues/185